tests/test_etcd3.py

```diff
@@ -101,6 +101,11 @@ def test_get_random_key(self, etcd, string):
         returned, _ = etcd.get('/doot/' + string)
         assert returned == b'dootdoot'
 
+    def test_get_have_cluster_revision(self, etcd, string):
```

Comment:
add

    @given(characters(blacklist_categories=['Cs', 'Cc']))

above this testcase (same as the previous two), otherwise pytest doesn't know what `string` is